src/Vault.sol

```diff
@@ -458,7 +452,7 @@ contract Vault is IVault, ERC20PermitUpgradeable, AccessControlUpgradeable, Reen
         returns (uint256, uint256)
```

Comment:
Related to the other computation

https://github.com/yieldnest/yieldnest-vault/blob/217749891802a3a459d81e5c9dc9941153f8e8d9/src/Vault.sol#L437

_convertToAssets doesn't seem correct as it would first need to convert the shares to base asset, and then convert that to asset_.

Right now the conversion `_convertAssetToBase(asset_, shares);` converts shares as though they are assets

Reply:
@danoctavian great, thanks for noticing this.
I pushed an update here kindly review when you can.